packages/curriculum-compiler-json/index.js

```diff
@@ -63,6 +63,31 @@ function compileInsight(ast) {
   return json;
 }
 
+function compileQuestion(ast) {
+  if (!ast || !ast.children) {
+    throw new Error('Missing or invalid AST');
```

Comment:
no need for `new`

Reply:
It's equivalent to `throw Error(...)`

Reply:
Maybe we should make that a lint rule?

Reply:
cc @Loopiezlol

Reply:
@nem035 I had actually written it that way at first, then saw that it was `throw new Error()` elsewhere and thought it'd be better to be consistent (without changing the rest of the codebase as part of this PR)